Makefile

```diff
@@ -71,6 +71,24 @@ push-contoller: build-controller
 	$(CONTAINER_CMD) tag cmd/manager:forklift-controller-image ${IMG}
 	$(CONTAINER_CMD) push ${IMG}
 
+# Build the docker image
+build-api:
+	bazel run cmd/forklift-api:forklift-api-image
+
+# Push the docker image
+push-api: build-api
+	$(CONTAINER_CMD) tag cmd/forklift-api:forklift-api-image ${API_IMAGE}
```

Comment:
we have: `IMG ?= quay.io/ocpmigrate/forklift-controller:latest`
shouldn't we have the same approach to the `API_IMAGE` and `OPERATOR_IMAGE`?

Reply:
yes, those changes sound good but unrelated to this PR, would you mind posting them separately?

Reply:
> yes, those changes sound good

I mean, changing IMG to CONTROLLER_IMAGE and replace ocpmigrate but I don't care much about quay.io, developers should go through quay.io..

Reply:
> yes, those changes sound good but unrelated to this PR, would you mind posting them separately?

yeah sounds ok.
the channel usage is asked in your other PR, once it's resolve it's LGTM